site/profile/templates/userportal/settings.py.epp

```diff
@@ -0,0 +1,276 @@
+from pathlib import Path
```

Comment:
We could define a settings_local.py imported by settings.py and avoid having in MC puppet code a bunch of variables that are not modified by the epp tempalte:
https://code.djangoproject.com/wiki/SplitSettings#Multiplesettingfilesimportingfromeachother